modules/gitops-release-notes-1-15-3.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes-1-15.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="gitops-release-notes-1-15-3_{context}"]
+= Release notes for {gitops-title} 1.15.3
+
+{gitops-title} 1.15.3 is now available on {OCP} 4.14, 4.15, 4.16, and 4.17.
+
+[id="errata-updates-1-15.3_{context}"]
+== Errata updates
+
+[id="RHSA-XXXX:NNNN-gitops-1-15-3-security-update-advisory_{context}"]
+=== RHSA-XXXX:NNNN - {gitops-title} 1.15.3 security update advisory
+
+Issued: 2025-06-28
+
+The list of security fixes that are included in this release is documented in the following advisory:
+
+* link:https://access.redhat.com/errata/RHSA-XXXX:NNNN[RHSA-XXXX:NNNN]
+
+If you have installed the {gitops-title} Operator in the default namespace, run the following command to view the container images in this release:
+
+[source,terminal]
+----
+$ oc describe deployment gitops-operator-controller-manager -n openshift-gitops-operator
+----
+
+[id="fixed-issues-1-15-3_{context}"]
+== Fixed issues
+
+* Before this update, the RBAC ConfigMap's scope was configured with `\\[groups,email\\]` but the corresponding `Spec.RBAC.Scopes` remained unchanged. This discrepancy led to conflicts during ConfigMap synchronization, specifically when transitioning to Keycloak SSO and modifying the Argo CD configuration. With this update, a new mechanism is implemented in the {gitops-title} Operator to maintain the existing RBAC scope configuration within the RBAC ConfigMap when upgrading to Keycloak SSO. link:https://issues.redhat.com/browse/GITOPS-5977[GITOPS-5977]
```

Comment:
```suggestion
* Before this update, the RBAC ConfigMap's scope was configured with `\\[groups,email\\]` but the corresponding `Spec.RBAC.Scopes` remained unchanged. This discrepancy led to conflicts during ConfigMap synchronization when transitioning to Keycloak SSO. With this update, a new mechanism is implemented in the {gitops-title} Operator to maintain the existing RBAC scope configuration within the RBAC ConfigMap when upgrading to Keycloak SSO. link:https://issues.redhat.com/browse/GITOPS-5977[GITOPS-5977]
```

Reply:
@Dhruv-Soni11 I changed a bit the text regarding the issue only pops up when updating to keycloak SSO.